lang/zh-TW.js

```diff
@@ -0,0 +1,227 @@
+    "support-link": {
+      "title": "以新頁籤開啟 micro:bit 支援中心",
+      "label": "支援"
+    },
+    "help-link": {
+      "title": "以新頁籤開啟此編輯器的幫助文件",
+      "label": "幫助"
+    },
+    "issues-link": {
+      "title": "在 GitHub 中檢視 Python 編輯器的未解決問題",
+      "label": "問題追蹤"
+    },
+    "feedback-link": {
+      "title": "給我們關於 Python 編輯器的回饋",
+      "label": "回饋意見"
+    },
+    "editor-ver": "編輯器版本：",
+    "mp-ver": "MicroPython 版本："
+  },
+  "confirms": {
+    "quit": "有些內容變更後未儲存，確定要離開嗎？",
+    "blocks": "你有未儲存的程式。使用積木模式會更改你的代碼，你可能會遺失這些變更，確定要繼續嗎？",
+    "replace_main": "添加的 main.py 檔案將取代編輯器中現有的程式碼！",
+    "replace_file": "要取代 \"{{file_name}}\" 這個檔案嗎？",
+    "replace_module": "要取代 \"{{module_name}}\" 這個模組嗎？",
+    "download_py_multiple": "這個專案裡有多個不使用此格式保存的檔案。\n我們建議下載成 Hex 檔案，它將包含整個專案，也能在編輯器上重新讀取。\n\n確定只要下載 {{file_name}} 這個檔案？"
+  },
+  "code": {
+    "start": "在此處添加 Python 程式碼。例如："
+  },
+  "webusb": {
+    "err": {
+      "update-req": "你需要<a target=\"_blank\" href=\"https://support.microbit.org/support/solutions/articles/19000019131-how-to-upgrade-the-firmware-on-the-micro-bit\">更新 micro:bit 韌體</a>以使用這個功能。",
+      "clear-connect": "另一個程序正在連線到這個設備。<br>關閉可能也在使用 WebUSB 的其它頁籤（像是 MakeCode、Python 編輯器），或者斷開 micro:bit 再重新連接。",
+      "reconnect-microbit": "請重新連接你的 micro:bit 後再試一次。",
+      "partial-flashing-disable": "如果錯誤仍然存在，請試著停用 Beta 選項中的快速燒錄功能。",
+      "device-disconnected": "設備已斷開連接。",
+      "timeout-error": "無法連接到 micro:bit",
+      "unavailable": "WebUSB 可以讓你直接在線上編輯器編程並連結序列埠監控台。<br/>可惜，這個瀏覽器不支援 WebUSB，建議你改用 Chrome 或是基於 Chrome 內核的其它瀏覽器。",
+      "find-more": "了解更多"
+    },
+    "troubleshoot": "疑難排解",
+    "close": "關閉",
+    "request-repl": "發送 CTRL-C 到 REPL",
+    "request-serial": "發送 CTRL-D 以重設",
+    "flashing-text": "燒錄 micro:bit",
+    "download": "下載 Hex 檔案"
+  },
+  "load": {
+    "show-files": "顯示檔案",
+    "load-title": "讀取",
+    "instructions": "拖曳一個 .hex 或 .py 檔案到這裡以開啟它。",
+    "submit": "讀取",
+    "save-title": "儲存",
+    "save-hex": "下載專案 Hex 檔案",
+    "save-py": "下載 Python 腳本",
+    "fs-title": "檔案",
+    "toggle-file": "或瀏覽裝置中的檔案。",
+    "fs-add-file": "新增檔案",
+    "hide-files": "隱藏檔案",
+    "td-filename": "檔案名稱",
+    "td-size": "大小",
+    "fs-space-free": "可用空間",
+    "remove-but": "移除",
+    "save-but": "儲存",
+    "files-title": "專案檔案",
+    "help-button": "檔案説明",
+    "file-help-text": "專案檔案區域顯示程式中引用（include）的檔案，並允許你添加或移除外部 python 模組和其它檔案。要進一步瞭解請參閱",
+    "help-link": "Python 編輯器幫助文件",
+    "invalid-file-title": "無效的檔案類型",
+    "mpy-warning": "這個版本的 Python 編輯器還不支援添加 .mpy 檔案。",
+    "extension-warning": "Python 編輯器只能載入副檔名為 .hex 或 .py 的檔案。"
+  },
+  "languages": {
+    "en": {
+      "title": "英文"
+    },
+    "es": {
+      "title": "西班牙语"
+    },
+    "pl": {
+      "title": "波蘭語"
+    },
+    "hr": {
+      "title": "克羅地亞語"
+    },
+    "zh-CN": {
+      "title": "中文（簡體）"
+    },
+    "zh-HK": {
+      "title": "Chinese (traditional, Hong Kong)"
+    },
+    "zh-TW": {
+      "title": "Chinese (traditional, Taiwan)"
+    }
```

Comment:
Should we for the translation to be finished?

Reply:
I think we would need this to be translated to resolve the language menu issue https://github.com/bbcmicrobit/PythonEditor/pull/316#discussion_r366362408 or we can add the language and wait for the translation to come at a later date

Reply:
Since this is going straight to master I think I'd prefer to make sure the translation is finished.

Reply:
This has now been updated